tremor-connectors/src/utils/tls.rs

```diff
@@ -216,167 +204,57 @@ pub enum Error {
 }
 
 /// Load the passed certificates file
-fn load_certs(path: &Path) -> Result<Vec<Certificate>, Error> {
-    let certfile = tremor_common::file::open(path)?;
+fn load_certs<'x>(path: &PathBuf) -> Result<Vec<CertificateDer<'x>>, Error> {
+    let certfile = File::open(path)?;
```

Comment:
I think we want to keep tremor_common ... iwe added it for hygernic errors as the errors from file itself are less then helpful to a user

Reply:
Oh snot. I totally missed that